src/fear_and_greed/cnn.py

```diff
@@ -26,7 +26,13 @@ class Fetcher:
     """Fetcher gets the HTML contents of CNN's Fear & Greed Index website."""
 
     def __call__(self) -> dict:
-        r = requests.get(URL)
+        headers = {
+            'User-Agent': 'Mozilla',
+        }
+        r = requests.get(URL, headers=headers)
+        if r.status_code != 200:
```

Comment:
As suggested in the linked ticket you could also use `r.raise_for_status()`.

Reply:
Yes, I switched to that approach in 25c9895adb654a20c35179524de6d42ddd0a47b4.

Reply:
Cool stuff. Thanks for the quick response and the v0.3 release to pypi.